clients/include/rocblas.hpp

```diff
@@ -155,26 +151,29 @@
         rocblas_operation transA, rocblas_diagonal diag,
         rocblas_int m, rocblas_int n,
         const T* alpha,
-        const T* A, rocblas_int lda,
-        T*       B, rocblas_int ldb);
+        T* A, rocblas_int lda,
```

Comment:
That is a hcc compiler bug. In the trsm code, I have  to use *float *ptr= A+offset",I believe it will pass in gcc. but in hcc, it reports an error. so I have to remove the const qualifier.